util/PostgresMigrations/Migrations/20241016140839_AddIsMultiOrgEnterpriseColumn.cs

```diff
@@ -0,0 +1,1739 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace Bit.PostgresMigrations.Migrations;
+
+/// <inheritdoc />
+public partial class AddIsMultiOrgEnterpriseColumn : Migration
```

Comment:
❌ Looks like something got messed up with your EF Migration here. It appears to be rebuilding the entire database in this file. Check out your [MySql migration file](https://github.com/bitwarden/server/pull/4903/files#diff-bd6759498181b48264485e5081fd40a5f9d754c4c0bcafa83f5ab288ddc8b707). That's what we want.